### PR TITLE
add case ocp-28140

### PR DESCRIPTION
**Pull request by QiaolingTang**
@anpingli PTAL, thanks
case: https://url.corp.redhat.com/e312b4b
runner-v3 log:  https://url.corp.redhat.com/3591f88

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
/lgtm

File: features/logging/elasticsearch.feature [hunk 1]
```diff
@@ -86,3 +86,63 @@ Feature: Elasticsearch related tests
...
+    When I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | _cat/indices?format=JSON |
+      | op           | GET                      |
+    Then the step should succeed
+    And evaluation of `@result[:parsed].select {|e| e['index'].start_with? "app"}.map {|x| x["index"]}` is stored in the :app_indices clipboard
+    And evaluation of `@result[:parsed].select {|e| e['index'].start_with? "infra"}.map {|x| x["index"]}` is stored in the :infra_indices clipboard
+    And evaluation of `@result[:parsed].select {|e| e['index'].start_with? "audit"}.map {|x| x["index"]}` is stored in the :audit_indices clipboard
+    
+    Given I successfully merge patch resource "elasticsearch/elasticsearch" with:
+      | {"spec": {"managementState": "Unmanaged"}} |
```
Comment (liangxia) [inline features/logging/elasticsearch.feature:122]: Do we need to restore the change after the test ?

Comment (QiaolingTang) [reply features/logging/elasticsearch.feature:122]: The CR will be removed after the scenario([line 108](https://github.com/openshift/verification-tests/pull/1664/files#diff-f444ec1a915696d4912ef14fc13d41becb90880d8cad5d4540fa9aaa85a85adcR108)), so no need to restore the changes.

Comment (liangxia) [reply features/logging/elasticsearch.feature:122]: Ack

**Pull request merged**